composer.json

```diff
@@ -8,10 +8,30 @@
             "email": "changeme@mailinator.com"
         }
     ],
-    "autoload" : {
-        "classmap": ["."]
```

Comment:
This was adding more than just the csstidy classes since it recursively scans all directories in the path specified, so this was picking up any classes in the `testing` directory, as well as the `vendor` directory (which was some composer classes).  I changed this to just the files that contain classes in the root directory (seen below).